ui/apps/platform/src/Containers/Policies/Table/PoliciesTable.tsx

```diff
@@ -465,15 +471,37 @@ function PoliciesTable({
                 </Table>
             </PageSection>
             <ConfirmationModal
+                title={`${
+                    isExternalPolicySelected(policies, deletingIds)
```

Comment:
Ouch, interaction design demotes **Permanently delete** to **Delete** even if some locally managed?